pkg/apis/extensions/v1beta1/types.go

```diff
@@ -242,6 +242,11 @@ type DeploymentSpec struct {
 	// No label is added if this is set to empty string.
 	UniqueLabelKey *string `json:"uniqueLabelKey,omitempty"`
 
+	// Minimum number of seconds for which a newly created pod should be ready
```

Comment:
Could you please put this field above RevisionHistoryLimit? I think it will be more frequently used, so I'd like to keep it closer to the top.